### PR TITLE
Feat: Added Buttons CSS

### PR DESCRIPTION
**Pull request by smilegupta**
Also in the PR
-> Added a few typographies
-> Added comments in the CSS files

### CODE REVIEW

File: styles/buttons.css [hunk 1]
```diff
@@ -35,6 +37,142 @@
   color: var(--primary-color);
 }
 
+.link-text-primary {
+  text-decoration-color: var(--primary-color);
+  text-decoration: underline;
```
Comment (Naveenchandar) [inline styles/buttons.css:42]: Use shorthand

Comment (smilegupta) [reply styles/buttons.css]: Done

File: styles/buttons.css [hunk 2]
```diff
@@ -35,6 +37,140 @@
...
+  opacity: 0.5;
+}
+
+/*************** Buttons with Different Sizes ***************/
+
+.btn-lg-size {
+  font-size: 1.5rem;
+  padding: 1rem 1.5rem;
+}
+.btn-sm-size {
```
Comment (rohanmathur91) [inline styles/buttons.css:103]:
``` suggestion
You can just give btn-lg and btn-sm as lg, sm represents size and btn says that its button, so no need to specifically mention size.
```

**Review by Naveenchandar — APPROVED** [review]
Looks good

File: styles/buttons.css [hunk 3]
```diff
@@ -35,6 +37,140 @@
...
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  position: fixed;
+  bottom: 1rem;
+  right: 1rem;
+  width: 50px;
+  height: 50px;
```
Comment (rohanmathur91) [inline styles/buttons.css:120]: Any specific reason to have px?

Comment (smilegupta) [reply styles/buttons.css:120]: nothing specific, just wanted to have a specific value for the floating button.. Any suggestions in mind? @rohanmathur91

File: styles/buttons.css (hunk 2 above)
Comment (smilegupta) [reply styles/buttons.css:103]: done

**Review by rohanmathur91 — APPROVED** [review]
Added a few suggestions you can ignore if you want, rest looks good.

File: styles/buttons.css (hunk 3 above)
Comment (rohanmathur91) [reply styles/buttons.css:120]: rems can be used.

**Pull request merged**